src/app/shared/refresh-organization/refresh-organization.component.html

```diff
@@ -17,7 +17,7 @@
 <button
   mat-button
   color="accent"
-  matTooltip="Refresh all entries in the organization"
+  matTooltip="Refresh all tools in the namespace"
```

Comment:
Based on the directory name `shared`, I think this is a shared component for tools and workflows. If so, you will need to have different messages for workflows and tools. And possibly services. Things aren't always as straightforward as they seem. :(

Reply:
We completely removed the refresh organization button from workflows (and services) so this might not actually be an issue. AFAIK only the tools ui is using this logic.

Reply:
Ah nvm I thought it was refresh all